src/EditorFeatures/Test2/FindReferences/FindReferencesTests.AliasSymbols.vb

```diff
@@ -214,6 +214,83 @@ using $$AliasToC = N.[|C|];
 
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("RuleCategory", "RuleId", Scope = "member", Target = "~M:N.[|C|].Goo")]
 
+namespace N
+{
+    class {|Definition:C|}
+    {
+        void Goo()
+        {
+        }
+    }
+}
+        </Document>
+    </Project>
+</Workspace>
+            Await TestAPIAndFeature(input, kind, host)
+        End Function
+
+        <WpfTheory, CombinatorialData, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        Public Async Function TestAliasReferenceInGlobalSuppression_WithAttributeSuffix(kind As TestKind, host As TestHost) As Task
+            Dim input =
+<Workspace>
+    <Project Language="C#" CommonReferences="true">
+        <Document>
+using $$AliasToC = N.[|C|];
+
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessageAttribute("RuleCategory", "RuleId", Scope = "member", Target = "~M:N.[|C|].Goo")]
```

Comment:
Can you add tests that don't use the fully qualified name to make sure we're getting coverage where `IsQualifiedName` is false?

Reply:
i did add those tests :)

Reply:
shhhh.... It's Friday 👀